packages/core/content-manager/server/services/entity-manager.js

```diff
@@ -224,9 +137,7 @@ module.exports = ({ strapi }) => ({
 
   // FIXME: handle relations
   deleteMany(opts, uid) {
-    const params = { ...opts };
```

Comment:
hurm, didn't know that could be the case 🤔 Still I don't think it could be an issue as we are not reusing opts somewhere else, and we are not also deepCopying options, so we might have mutations

Reply:
Yeah I think we can drop it too, just wanted to add a warning